phpstan.neon.dist

```diff
@@ -0,0 +1,19 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    level: 0
+    bootstrapFiles:
+        - vendor/bin/.phpunit/phpunit/vendor/autoload.php
```

Comment:
With this, there is something to add to the composer.json

https://github.com/sonata-project/SonataAdminBundle/blob/master/composer.json#L117-L124

But maybe we should start doing: 
https://github.com/sonata-project/dev-kit/issues/1559

Reply:
Agree on the dev-kit issue, but I would like not to block this one until a decision is made. So for now I just added the composer thing